### PR TITLE
Migrate envtool to Kong

### PR DESCRIPTION
**Pull request by noisersup**
# Description

This PR is a part of #1156.

## Readiness checklist

* [x] I added tests for new functionality or bugfixes.
* [x] I ran `task all`, and it passed.
* [x] I added/updated comments for both exported and unexported top-level declarations (functions, types, etc).
* [ ] I checked comments rendering with `task godocs`.
* [x] I ensured that the title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

File: cmd/envtool/main.go [hunk 1]
```diff
@@ -277,18 +277,17 @@ func run(ctx context.Context, logger *zap.SugaredLogger) error {
 	return nil
 }
 
-func main() {
-	debugF := flag.Bool("debug", false, "enable debug mode")
-	flag.Parse()
+// The cli struct represents all command-line commands, fields and flags.
```
Comment (rumyantseva) [inline cmd/envtool/main.go:280]:
```suggestion
// cli struct represents all command-line commands, fields and flags.
```

to make it like everywhere (not super critical I guess).

**Review by rumyantseva — APPROVED** [review]
LGTM overall.

**Comment by AlekSi** [comment]
(envtool is an internal tool, so that's chore; FerretDB binary flags are user-visible, so PR for that would be a feature)

**Review by AlekSi — CHANGES_REQUESTED** [review]
> I marked all done items in this checklist.

That item is unchecked, so I assume that wasn't done

**Comment by noisersup** [comment]
@AlekSi Sorry, it's marked now

**Review by rumyantseva — APPROVED** [review]
LGTM

**Review by AlekSi — APPROVED** [review]

**Pull request merged**